Makefile

```diff
@@ -53,7 +67,7 @@ unit: kubebuilder
 KUBEBUILDER_ASSETS_ERR := not detected in $(KUBEBUILDER_ASSETS), to override the assets path set the KUBEBUILDER_ASSETS environment variable, for install instructions see https://pkg.go.dev/sigs.k8s.io/controller-runtime/tools/setup-envtest
```

Comment:
It would be great to change this to do a similar automatic setup from `setup-envtest`  like we do in operator-controller.

Reply:
Yup yup - already have that in some commit in here somewhere. Actually, I think I might remove bingo from this PR, then add it as its own. To keep this change even smaller.